tracpro/baseline/views.py

```diff
@@ -144,9 +150,27 @@ def form_valid(self, form):
                     submitted_on=baseline_datetime,
                     category=u'')
 
+        def form_valid(self, form):
+            baseline_question = self.form.cleaned_data['baseline_question']
+            follow_up_question = self.form.cleaned_data['follow_up_question']
+            contacts = self.form.cleaned_data['contacts']
+            baseline_minimum = self.form.cleaned_data['baseline_minimum']
+            baseline_maximum = self.form.cleaned_data['baseline_maximum']
+            follow_up_minimum = self.form.cleaned_data['follow_up_minimum']
+            follow_up_maximum = self.form.cleaned_data['follow_up_maximum']
+            start = self.form.cleaned_data['start_date']
+            end = self.form.cleaned_data['end_date']
+
+            # Create a single PollRun for the Baseline Poll
+            self.create_baseline(baseline_question.poll, start, contacts, baseline_question,
```

Comment:
It looks like this could be moved into the for loop if you remove the `loop_count > 0` conditional.